File: simple_apk/src/com/example/screenshotocr/MainActivity.java
package com.example.screenshotocr;

import android.app.Activity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        
        Button btnScan = findViewById(R.id.btnScan);
        Button btnSearch = findViewById(R.id.btnSearch);
        
        btnScan.setOnClickListener(v -> 
            Toast.makeText(this, "Scan feature coming soon!", Toast.LENGTH_SHORT).show()
        );
        
        btnSearch.setOnClickListener(v -> 
            Toast.makeText(this, "Search feature coming soon!", Toast.LENGTH_SHORT).show()
        );
    }
}

File: app/src/main/java/com/example/screenshotocr/MainActivity.kt
package com.example.screenshotocr

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnScan = findViewById<Button>(R.id.btnScan)
        val btnSearch = findViewById<Button>(R.id.btnSearch)

        btnScan.setOnClickListener {
            Toast.makeText(this, "OCR Scan feature - Coming Soon!", Toast.LENGTH_SHORT).show()
        }

        btnSearch.setOnClickListener {
            Toast.makeText(this, "Search feature - Coming Soon!", Toast.LENGTH_SHORT).show()
        }
    }
}